presto-tests/src/test/java/com/facebook/presto/tests/TestSpillingQueries.java

```diff
@@ -24,12 +24,12 @@
 import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
 import static com.facebook.presto.tpch.TpchMetadata.TINY_SCHEMA_NAME;
 
-public class TestDistributedSpilledQueries
+public class TestSpillingQueries
```

Comment:
> The fact it is using distributed query runner it is not so much important here.

The fact that it is _not using_ the local runner _is_ important. Memory revocation is emulated in local query runner.